enclave_wrangler/main.py

```diff
@@ -217,9 +234,34 @@ def run(input_csv_folder_path):
         # noinspection PyUnusedLocal
         # add expressionItems to version -----
         response_json = post_request_enclave_api_addExpressionItems(header, upd_cs_ver_expression_items_dict)
-        print(json.dumps(response_json))
+        print('post request to add expressionItems returned: ----------' + json.dumps(response_json))
+        # Once the expression items has been added save the enclave concept_id so that we can update the code_sets.csv file
+        # update code_sets_df with the enclave_codeset_id column of the  value in the codeset_id retured from the enclave
+        # and if needed we can also save the json data in upd_cs_ver_expression_items_dict
+        # premade_codeset_id is stored in the codeset_id column in the csv files, save the id in the enclave_codeset_id column
+        # update when it was uploaded as well, Stephane 3/15/22
+        code_sets_df.set_index('codeset_id', inplace=True)
+        code_sets_df.at[premade_codeset_id, 'enclave_codeset_id'] = codeset_id
+        code_sets_df.at[premade_codeset_id, 'enclave_codeset_id_updated_at'] = _datetime_palantir_format()
+        code_sets_df.reset_index()
         # return response_json
 
 
+    # write out the update csv file with the enclave_codeset_id
+    # print('before terminating write out the updated code_sets.csv file here')
+    # date_str = datetime.now().strftime('%Y_%m_%d_%H_%M')
+    # output_filename = 'code_sets_updated_' + date_str + '.csv'
+    output_filename = 'code_sets.csv'
+    code_sets_df.to_csv(os.path.join(input_csv_folder_path, output_filename), index=True, encoding='utf-8')
+
+    # TODO:
+    persistence_csv_path = os.path.join(PROJECT_ROOT, 'data', 'cset.csv')
+    code_sets_df_limited = code_sets_df[['enclave_codeset_id', 'enclave_codeset_id_updated_at', 'concept_set_name']]
+    persistence_df = pd.read_csv(persistence_csv_path).fillna('')
+    persistence_df_new = persistence_df.merge(
```

Comment:
@Sigfried Stephanie wrote some code to save some of the assigned enclave IDs in a specific file. I updated this code to also update the file that we're currently mainly using for persistence, `cset.csv`.

Reply:
@Sigfried Oh, also I saw that the columns `grouped_by_bids` and `concept_id` were empty in `cset.csv`, and I don't think they were being used, so I deleted them.